lib/handleRequest.js

```diff
@@ -6,7 +6,13 @@ const validateSchema = validation.validate
 const { hookRunner, hookIterator } = require('./hooks')
 const wrapThenable = require('./wrapThenable')
 
-function handleRequest (request, reply) {
+function handleRequest (err, request, reply) {
+  if (reply.sent === true) return
+  if (err) {
```

Comment:
Can you add a comment why are we receiving an error here? can you make the check explicit `err !== null && err !== undefined`?

Reply:
Ok, I'll update also the rest of the codebase.